src/github.com/couchbase/sync_gateway/db/sequence_id.go

```diff
@@ -16,13 +16,28 @@ import (
 // that gave the user access.
 type SequenceID struct {
 	TriggeredBy uint64 // The sequence # that triggered this (0 if none)
+	LowSeq      uint64 // Lowest contiguous sequence seen on the feed
 	Seq         uint64 // The actual internal sequence
 }
 
 var MaxSequenceID = SequenceID{Seq: math.MaxUint64}
 
+// Sequence IDs can be in one of the following formats:
+//   Seq                    - simple sequence
+//   TriggeredBy:Seq        - when TriggeredBy is non-zero, LowSeq is zero
+//   LowSeq:TriggeredBy:Seq - when LowSeq is non-zero.
+// When LowSeq is non-zero but TriggeredBy is zero, will appear as LowSeq::Seq.
+// When LowSeq is non-zero but is less than s.Seq (occurs when sending previously skipped sequences), ignore LowSeq.
```

Comment:
I think in that case, these comments probably belong on SequenceID.String(), and maybe just reference that here by saying "For rules on sequence formats, see SequenceID.String()"

This makes sense:

> The idea is to only send the low sequence when it's meaningful. Sending sequence like "5::8" to the client really means "Here's sequence 8, but you might be missing sequences between 5 and 8". We don't want to send in this format when backfilling an older sequence. e.g. when sending sequence 3 to a client, we don't want to send as 5::3.

but in light of that, the original comment:

>  When LowSeq is non-zero but is less than s.Seq (occurs when sending previously skipped sequences), ignore LowSeq.

is confusing, because I thought the LowSeq should be ignored (left off when formatted) when LowSeq is *greater* than S.Seq (in your example above, LowSeq=5 and S.Seq=3)